### ISSUE TITLE
Update gradle files section in (Android Room with a View - Kotlin)

### ISSUE THREAD

**Issue by vigneshshettyin** [googlecodelabs/android-room-with-a-view#178]
Reference Link: https://developer.android.com/codelabs/android-room-with-a-view-kotlin#3

According to the docs given following snippet must be added to app-level graddle `apply plugin: 'kotlin-kapt'`. This doesn't seem to work in a newer version of the android studio the above step should be replaced by 
```
plugins {
    id 'com.android.application'
    id 'kotlin-android'
    id 'kotlin-kapt'
}
```

Screenshot attached for reference

![Capture](https://user-images.githubusercontent.com/77713888/120929069-25ed3a80-c705-11eb-9d83-9e15eb54c3b5.PNG)

**Comment by florina-muntenescu** [comment]
I'm using Android Studio Arctic Fox beta03 and this seems to work. 
What error are you getting and which Android Studio version are you using?

**Comment by vigneshshettyin** [comment]
Hey, @florina-muntenescu thanks for the reply.
btw, I am a beginner in Android Development. I am currently using Android Studio Stable release 4.2.1.  
![Screenshot 2021-06-13 194324](https://user-images.githubusercontent.com/77713888/121810757-aff64f80-cc7f-11eb-8f12-45741448dbd8.png)

The steps given in codelabs didn't match my android studio gradle configuration. 

ie
![Screenshot 2021-06-13 194649](https://user-images.githubusercontent.com/77713888/121810900-22672f80-cc80-11eb-8af1-e74a890feabc.png)

After referring to some articles and blogs I got to know same gradle configuration can be written in different ways.  I just had a suggestion as in most codelabs visitors will be beginners it would be great to keep it updated to the latest version. Correct me if I am wrong.

**Comment by florina-muntenescu** [comment]
Fixed in #181 . Thanks for reporting!
